modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/resources/com/liferay/portal/workflow/metrics/internal/search/index/dependencies/workflow-metrics-update-task-script.painless

```diff
@@ -1,10 +1,11 @@
 if (ctx._source.containsKey("tasks")) {
-	def task = ctx._source.tasks.find(task -> task.taskId == params.taskId);
+	def taskParam = params.task;
+	def task = ctx._source.tasks.find(task -> task.taskId == taskParam.taskId);
```

Comment:
HEy @MarinhoFeliphe, I think, it is not required to create the temp variable `taskParam`. We could use `params.task` expression instead of this(like we use the `ctx._source.tasks`). 

What do you think?

Thx

Reply:
It's better!